git_machete/utils.py

```diff
@@ -295,3 +296,24 @@ def format_choice(c: str) -> str:
         else:
             return colored(c, AnsiEscapeCodes.ORANGE)
     return f" ({', '.join(map_truthy_only(format_choice, choices))}) "
+
+
+class EdgeColor(Enum):
+    DIM = auto()
```

Comment:
The (abstract) edge color is `GRAY`... it just happens to be implemented by `DIM` coloration in text terminal (but in IntelliJ plugin, it's implemented with the actual gray color)